### ISSUE TITLE
Karma and Todos for Yonatan Zarecki

### ISSUE THREAD

**Issue by zvili** [TechnionYP5777/Bugquery#5]
_From @yonzarecki on October 30, 2016 10:8_

- [x]  Use a picture, photo or distinct avatar. Don''t use an automatic icon
- [x]  Make sure you present yourself by a readable nick
- [x] Describe yourself in the bio page
- [x] State that you are (not) in search for a job

_Copied from original issue: SpartanRefactoring/Spartanizer#613_

**Comment by zvili** [comment]
_From @yosefraisman on October 30, 2016 12:12_

Something about your karma page really speaks to me, @yonzarecki. Please tell me what you think about #616

**Comment by zvili** [comment]
_From @AviadCo on October 30, 2016 18:26_

Great Job :) You can view mine if you want: #528

**Comment by zvili** [comment]
_From @ZivIzhar on October 30, 2016 19:38_

great page : D
here's mine : #644

**Comment by zvili** [comment]
_From @yonzarecki on October 30, 2016 19:40_

Done. @yossigil

**Comment by zvili** [comment]
_From @yossigil on October 31, 2016 7:37_

:100:

**Comment by yossigil** [comment]
Can you summarize here all issues you dealt with towards Sprint 1?

@AdiOmari

**Comment by yonzarecki** [comment]
@yossigil 
I've closed issue #3 . 
As stated in roded's karma page #10,  I'm also working on #8, #14, #20 with @rodedzats. 

Our mini-team's job is bugquery's core algorithms such as the stack-trace comparison algorithm. Another job we've done is trying to organize the server-side code inside a single project so we and @ZivIzhar & @tonylekhtman will be able to collaborate while further building bugQuery's backend.

**Comment by yonzarecki** [comment]
This week I've worked on optimizing the code in both performance and design, as we agreed in the meeting.
I've worked on and closed issues #55, #56 about these matters.
After finishing my work, I've joined up with @ZivIzhar and helped with #53 where I could, while we worked we noticed about the new spartanizer version and opened #57.
I've also tried to help @tonylekhtman by giving him access to the computer in my office, from which he was able to get better computing power when working with the server (while it worked).

Right now I'm working on some cleanups for the Github repo starting with #60 and managing input better with #59 along with @rodedzats.

**Comment by AdiOmari** [comment]
@yonzarecki, Yonatan please choose a time for the project concluding meeting [here](https://docs.google.com/document/d/1UvP8tNBX4N-4dRM6ZIFREnbjf9Yvwo3TecFZJmJoApg/edit?usp=sharing)

**Comment by yonzarecki** [comment]
@AdiOmari Done.

**Comment by AdiOmari** [comment]
@ZivIzhar @yonzarecki Just make sure you set times in the same day.
